api/src/paths/project/{projectId}/survey/{surveyId}/templates/list.test.ts

```diff
@@ -0,0 +1,184 @@
+import chai, { expect } from 'chai';
+import { describe } from 'mocha';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import * as templateObservations from './list';
+import * as db from '../../../../../../database/db';
+import * as template_observations_queries from '../../../../../../queries/occurrence/template-observation-queries';
+import SQL from 'sql-template-strings';
+
+chai.use(sinonChai);
+
+describe('lists the survey template occurrences', () => {
```

Comment:
nit but this is so it makes it easier to search for tests associated to specific function later:

```suggestion
describe('getTemplateObservations', () => {
```